### PR TITLE
[hi] Localize security-chaos-engineering.md

### PR DESCRIPTION
**Pull request by abhay-raj19**
Signed-off-by : abhay-raj19 [rajpootabhay423@gmail.com](mailto:rajpootabhay423@gmail.com)
### Describe your changes
Localised a security chaos engineering document in hindi.
Ref: #2057 
As this PR was being successfully reviewed by @jayesh-srivastava and was waiting for second Reviewer.
But due to some changes from my end this PR got deleted suddenly.
And re-open feature was not being available from my end .

### Related issue number or link (ex: `resolves #issue-number`)
#1986 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
/lgtm , what about others!

File: content/hi/security-chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+को त्वरित रूप से पुनर्स्थापित करें जिससे जीरो डाउनटाइम और व्यवसाय पर असर को कम करने का लक्ष्य होता है।
+SREs और साइबर सुरक्षा इंजीनियर दोनों पूर्व-विफलता और उसके बाद के घटनाओं से निपटते हैं।
+अधिकांश सुरक्षा मुद्दे जल्दी से खोजने और ठीक करने में चुनौतीपूर्ण होते हैं, जो एप्लिकेशन या सिस्टम के कार्यक्षमता पर असर डालते हैं।
+साथ ही, सुरक्षा घटनाएं विकास चरण के दौरान खोजना आमतौर पर मुश्किल होता है।
+
+
+## समाधान
+
+सुरक्षा अव्यवस्था इंजीनियरिंग [अवलोकनीयता](/observability/) और साइबर प्रतिरोध क्षमता अभ्यासों के आधार पर बनाई गई है।
+इसका उद्देश्य "अज्ञात अज्ञात" को खोलना है और सिस्टम में आत्मविश्वास बढ़ाना है,
```
Comment (bishal7679) [inline content/hi/security-chaos-engineering.md:31]:
```suggestion
इसका उद्देश्य "अज्ञात अज्ञात" को उजागर करना और सिस्टम में आत्मविश्वास बनाना है,
```

File: content/hi/security-chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## समस्या
+
+
+[साइट अवारी इंजीनियर](/site-reliability-engineering/)(SREs) और साइबर सुरक्षा इंजीनियरों के लिए मुख्य प्राथमिकता होती है कि वे सेवा
+को त्वरित रूप से पुनर्स्थापित करें जिससे जीरो डाउनटाइम और व्यवसाय पर असर को कम करने का लक्ष्य होता है।
+SREs और साइबर सुरक्षा इंजीनियर दोनों पूर्व-विफलता और उसके बाद के घटनाओं से निपटते हैं।
+अधिकांश सुरक्षा मुद्दे जल्दी से खोजने और ठीक करने में चुनौतीपूर्ण होते हैं, जो एप्लिकेशन या सिस्टम के कार्यक्षमता पर असर डालते हैं।
+साथ ही, सुरक्षा घटनाएं विकास चरण के दौरान खोजना आमतौर पर मुश्किल होता है।
```
Comment (bishal7679) [inline content/hi/security-chaos-engineering.md:25]:
```suggestion
इसके अतिरिक्त, विकास चरण के दौरान सुरक्षा घटनाओं को उजागर करना आमतौर पर मुश्किल होता है।
```

File: content/hi/security-chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## समाधान
+
+सुरक्षा अव्यवस्था इंजीनियरिंग [अवलोकनीयता](/observability/) और साइबर प्रतिरोध क्षमता अभ्यासों के आधार पर बनाई गई है।
+इसका उद्देश्य "अज्ञात अज्ञात" को खोलना है और सिस्टम में आत्मविश्वास बढ़ाना है,
+साइबर प्रतिरोध क्षमता बढ़ाकर और अवलोकनीयता को सुधारकर।
+
+इंजीनियरिंग टीम धीरे-धीरे सुरक्षा संबंधी चिंताओं के लिए जागरूकता में सुधार करेंगे।
+जटिल बुनियादी संरचना, प्लेटफॉर्म और वितरित सिस्टमों के अंदर।
```
Comment (bishal7679) [inline content/hi/security-chaos-engineering.md:35]:
```suggestion
इंजीनियरिंग टीम धीरे-धीरे सुरक्षा संबंधी चिंताओं के लिए जागरूकता में सुधार करेंगे
जटिल बुनियादी संरचना, प्लेटफॉर्म और वितरित सिस्टमों के अंदर।
```

File: content/hi/security-chaos-engineering.md (hunk 2 above)
Comment (bishal7679) [reply content/hi/security-chaos-engineering.md:25]: as previously we were transliterating `development` as `डेवलपमेंट` What should we use for this case
/cc @Garima-Negi @jayesh-srivastava

**Review by bishal7679 — COMMENTED** [review]
Left some suggestions!
commit if you find its useful :+1: 
Let's have other approvers suggestions as well

File: content/hi/security-chaos-engineering.md (hunk 3 above)
Comment (kumarankit999) [reply content/hi/security-chaos-engineering.md:35]: Kindly Update all the changes as suggested by @bishal7679

**Comment by abhay-raj19** [comment]
@bishal7679 now check up on those changes, have corrected them .
and do review it again

**Comment by abhay-raj19** [comment]
Pls do review up the changes if you are satisfied with the changes do approve the request from your end @bishal7679

File: content/hi/security-chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+title: सुरक्षा अव्यवस्था इंजीनियरिंग  (Security Chaos Engineering)
+status: Completed
+category: अवधारणा
+tags: ["security", "methodology", ""]
+---
+
+## यह क्या है
+
+
+सुरक्षा अव्यवस्था इंजीनियरिंग(SCE) एक ऐसी शाखा है जो की [सीमावर्ती प्रणाली](/chaos-engineering/) पर आधारित है।
```
Comment (bishal7679) [inline content/hi/security-chaos-engineering.md:11]:
```suggestion
सुरक्षा अव्यवस्था इंजीनियरिंग(SCE) एक ऐसी शाखा है जो की [अव्यवस्था इंजीनियरिंग](/chaos-engineering/) पर आधारित है।
```

**Review by bishal7679 — CHANGES_REQUESTED** [review]
nits: just remove the extra lines L 10, 16, 19, 26

**Comment by abhay-raj19** [comment]
@kumarankit999 @bishal7679 @aj11anuj @jayesh-srivastava , made the changes as suggested now review it

File: content/hi/security-chaos-engineering.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+## समाधान
+
+सुरक्षा अव्यवस्था इंजीनियरिंग [अवलोकनीयता](/observability/) और साइबर प्रतिरोध क्षमता अभ्यासों के आधार पर बनाई गई है।
+इसका उद्देश्य "अज्ञात अज्ञात" को उजागर करना और सिस्टम में आत्मविश्वास बनाना है,
+साइबर प्रतिरोध क्षमता बढ़ाकर और अवलोकनीयता को सुधारकर।
+इंजीनियरिंग टीम धीरे-धीरे सुरक्षा संबंधी चिंताओं के लिए जागरूकता में सुधार करेंगे
+जटिल बुनियादी संरचना, प्लेटफॉर्म और वितरित सिस्टमों के अंदर।
+एससीई पूरे उत्पाद की साइबर प्रतिरोधकता में सुधार करता है, छिपी सुरक्षा समस्याओं को उजागर करता है,
+क्लासिकल ब्लाइंड स्पॉट को उजागर करता है और टीमों को महत्वपूर्ण एज केस के लिए तैयार करता है। 
+यह दृष्टिकोण [एसआरईएस(SREs)](/site-reliability-engineering/), [डेवऑप्स(DevOps)](/devops/) और [डेवसेकओप्स(DevSecOps)](/devsecops/) इंजीनियरों को सिस्टम में आत्मविश्वास बनाने,
```
Comment (kumarankit999) [inline content/hi/security-chaos-engineering.md:33]: यह दृष्टिकोण [एसआरईएस(SREs)](/site-reliability-engineering/), [डेवऑप्स(DevOps)](/devops/) और [डेवसेकओप्स(DevSecOps)](/devsecops/) इंजीनियरों को सिस्टम में विश्वास  बनाने,

**Review by kumarankit999 — COMMENTED** [review]
looks fixed now, what other approvers say??

**Comment by bishal7679** [comment]
pls remove L23

**Comment by abhay-raj19** [comment]
now fixed the suggested changes pls do check from your end @kumarankit999 @bishal7679

**Review by bishal7679 — APPROVED** [review]
Thank you @abhay-raj19 appreciate your effort!

**Comment by bishal7679** [comment]
Approver while merging make sure to apply squash and merge!

**Review by kumarankit999 — APPROVED** [review]
Thanks for your efforts!

**Pull request merged**